paddlets/automl/optimize_runner.py

```diff
@@ -109,18 +109,18 @@ def optimize(self,
                  in_chunk_len: int,
                  out_chunk_len: int,
                  train_tsdataset: Union[TSDataset, List[TSDataset]],
-                 valid_tsdataset: Union[TSDataset, List[TSDataset]] = None,
-                 sampling_stride: int = 1,
-                 skip_chunk_len: int = 0,
+                 valid_tsdataset: Optional[Union[TSDataset, List[TSDataset]]] = None,
+                 sampling_stride: Optional[int] = 1,
+                 skip_chunk_len: Optional[int] = 0,
```

Comment:
Line113 and Line 114 should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional